django/contrib/admin/templates/admin/change_form.html

```diff
@@ -73,15 +73,39 @@
             $(document).ready(function() {
                 $('.add-another').click(function(e) {
                     e.preventDefault();
-                    showAddAnotherPopup(this);
+                    var event = $.Event('django:add-another-related');
+                    $(this).trigger(event);
+                    if (!event.isDefaultPrevented())
+                        showAddAnotherPopup(this);
                 });
                 $('.related-lookup').click(function(e) {
                     e.preventDefault();
-                    showRelatedObjectLookupPopup(this);
+                    var event = $.Event('django:lookup-related');
+                    $(this).trigger(event);
+                    if (!event.isDefaultPrevented())
+                        showRelatedObjectLookupPopup(this);
                 });
+                $('body').on('click', '.related-widget-wrapper-link', function(e) {
```

Comment:
The difference is that the former will delegate the event attach to the body so new dom element with that class will get covered while the second will attach only to dom elements seen at document ready. I think the former behaviour is requested for at least inlines. but i haven't checked.